go-packages/meep-redis/db.go

```diff
@@ -112,6 +112,12 @@ func (rc *Connector) DBFlush(module string) error {
 	return nil
 }
 
+// EntryExists - true if entry exists; false otherwise
+func (rc *Connector) EntryExists(key string) bool {
+        value := rc.client.Exists(key).Val()
+        return value != 0
+}
+
```

Comment:
Please use goimports formatting on this change.

Reply:
ran and fixed alignment